package.json

```diff
@@ -7,7 +7,7 @@
   "main": "dist/umd/webex-components.umd.min.js",
   "scripts": {
     "build": "rollup -c",
-    "linter": "eslint src/ --ext .js,.jsx",
+    "linter": "eslint src/",
```

Comment:
Do we need this change?

Reply:
We do not need to specify extension separately